### DISCUSSION TITLE
Welcome to teste Discussions!

### DISCUSSION THREAD

**Discussion by toptoptuns** [toptoptuns/teste#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by toptoptuns** [comment]
minha é duvida é como subir um projeto react pelo codesandbox

**Reply by marehead** [reply]
você tem que logar sua conta do github no codesandbox !

**Comment by toptoptuns** [comment]
alguem conhece algum site para treinar html e css e js?

**Reply by marehead** [reply]
codepen!
